### PR TITLE
reordered imports

### PR DESCRIPTION
**Pull request by srbcheema1**
this PR fixes the #57 issue .
i have reordered the imports in lexicographical order .
Signed off by srbcheema1

### CODE REVIEW

File: scanapp/admin.py [hunk 1]
```diff
@@ -24,15 +24,15 @@
 from django.contrib import admin
 
 # Import models from scanapp.models
-from scanapp.models import Scan
-from scanapp.models import ScannedFile
-from scanapp.models import License
 from scanapp.models import Copyright
```
Comment (singh1114) [inline scanapp/admin.py:27]: Use it as follows:
```
from scanapp.models import (
    Copyright, CopyrightAuthor .....
)
```

File: scanapp/serializers.py [hunk 2]
```diff
@@ -25,15 +25,15 @@
 
 from rest_framework import serializers
 
-from scanapp.models import Scan
-from scanapp.models import ScannedFile
-from scanapp.models import License
 from scanapp.models import Copyright
```
Comment (singh1114) [inline scanapp/serializers.py:28]: Similar things will work.

File: scanapp/urls.py [hunk 3]
```diff
@@ -26,11 +26,11 @@
 from django.views.generic import TemplateView, RedirectView
 
 from scanapp.views import LocalUploadView
-from scanapp.views import ScanResults
-from scanapp.views import UrlScanView
-from scanapp.views import RegisterView
 from scanapp.views import LoginView
```
Comment (singh1114) [inline scanapp/urls.py:29]: Same with this. You can dig more files and use the same procedure.

**Review by singh1114 — CHANGES_REQUESTED** [review]

File: scanapp/urls.py [hunk 4]
```diff
@@ -25,12 +25,9 @@
...
 from django.views.generic import TemplateView, RedirectView
 
-from scanapp.views import LocalUploadView
-from scanapp.views import ScanResults
-from scanapp.views import UrlScanView
-from scanapp.views import RegisterView
-from scanapp.views import LoginView
-from scanapp.views import ScanApiView
+from scanapp.views import (
+    LocalUploadView, LoginView, RedirectView, ScanApiView, UrlScanView
```
Comment (singh1114) [inline scanapp/urls.py]: It's `RegisterView` not `RedirectView`.

**Review by singh1114 — CHANGES_REQUESTED** [review]

**Comment by srbcheema1** [comment]
@singh1114 that was my typo mistake. I have applied the changes requested and amended into last commit. please review

**Comment by singh1114** [comment]
Check why travis-CI is not working.

**Comment by srbcheema1** [comment]
@singh1114 travis-CI tests passed, please review

**Review by singh1114 — APPROVED** [review]

**Comment by singh1114** [comment]
@srbcheema1 Good work! I have approved the changes. You have to wait for these to be approved by @pombredanne.

**Comment by srbcheema1** [comment]
thanks @singh1114 I would love to contribute more.

**Comment by singh1114** [comment]
@srbcheema1  Surely. You can grab the new issue that I opened.

**Comment by pombredanne** [comment]
@srbcheema1 can you add a sign off to your commits or at least to the PR comment?

File: scanapp/admin.py [hunk 5]
```diff
@@ -24,15 +24,10 @@
...
-from scanapp.models import Copyright
-from scanapp.models import CopyrightHolder
-from scanapp.models import CopyrightStatement
-from scanapp.models import CopyrightAuthor
-from scanapp.models import Package
-from scanapp.models import ScanError
+from scanapp.models import (
+    Copyright, CopyrightAuthor, CopyrightHolder, CopyrightStatement, License,
+    Package, Scan, ScanError, ScannedFile
+)
```
Comment (pombredanne) [inline scanapp/admin.py]: I am sorry but I do not like this style of imports at all, can you use the same style we use elsewhere?

File: scanapp/serializers.py [hunk 6]
```diff
@@ -25,15 +25,10 @@
...
-from scanapp.models import Copyright
-from scanapp.models import CopyrightHolder
-from scanapp.models import CopyrightStatement
-from scanapp.models import CopyrightAuthor
-from scanapp.models import Package
-from scanapp.models import ScanError
+from scanapp.models import (
+    Copyright, CopyrightAuthor, CopyrightHolder, CopyrightStatement, License,
+    Package, Scan, ScanError, ScannedFile
+)
```
Comment (pombredanne) [inline scanapp/serializers.py]: Same comment as above, one import per line is what I prefer,

**Review by pombredanne — CHANGES_REQUESTED** [review]
Thank you! Please see my comments

**Comment by srbcheema1** [comment]
@pombredanne  I have made the requested changes. Please have a look

**Comment by pombredanne** [comment]
Thanks! do you mind to squash these commits in one?

**Comment by singh1114** [comment]
@pombredanne We can squash while merging.

**Comment by srbcheema1** [comment]
@pombredanne  please have a look at it

**Comment by pombredanne** [comment]
Thanks! can you add your signoff and reword your message to use the imperative style? See https://github.com/nexB/aboutcode/wiki/Writing-good-commit-messages

**Comment by pombredanne** [comment]
Much better! imperative style for the subject line would be instead: `Reorder imports` and `Remove trailing white spaces` instead... think of it like if you are giving an order to the codebase to update thyself!

**Comment by srbcheema1** [comment]
@pombredanne I will take care of naming in future. Thanks for this.

**Comment by pombredanne** [comment]
LGTM. Thank you!

**Comment by srbcheema1** [comment]
Thanks @pombredanne 
I am glad to see my first PR merged in this organisation. 
I would love to contribute more in this organization.

**Pull request merged**